src/torrent.cpp

```diff
@@ -8453,7 +8447,8 @@ namespace libtorrent
 			m_seeding_time += m_ses.session_time() - m_became_seed;
 
 		if (is_finished())
-			m_finished_time += m_ses.session_time() - m_became_finished;
+			m_finished_time += duration_cast<seconds>(aux::time_now()
```

Comment:
I'm not sure what you intended to do. but ``m_finished_time`` is a duration, ``time_now()`` is a time_point. you cannot add a time_point to a duration. If you think about it, adding two absolute times together is non-sensical.